backend/src/middlewares/validation/customRules/schemas/fishtank.js

```diff
@@ -0,0 +1,23 @@
+const { Fishtank, FishtankStatus } = require('../../../../database/models/index');
+
+/* Schemas don't support multiple custom rules yet so
+ * these are written in the right format for schemas
+ * but are currently used in the common methods
+ * (see ../../fishtank.js)
+ */
+module.exports = {
+  exists: {
+    options: async (value) => {
+      const fishtank = await Fishtank.findByPk(value);
+      return fishtank !== null;
+    },
+    errorMessage: 'doesn\'t exist',
```

Comment:
You can use `"` or ` to avoid escaping 😉

Reply:
Oh I thought the air-bnb ESLint would prevent me to do so